tests/e2e/utils/components.js

```diff
@@ -56,4 +56,28 @@ const loginAndConfirmExtensionActivation = async () => {
 	await page.waitForSelector( `tr[data-slug="${ slug }"] .deactivate a` );
 };
 
-export { createSimpleProduct, loginAndConfirmExtensionActivation };
+const deleteAllPackages = async () => {
+	await StoreOwnerFlow.login();
+	await StoreOwnerFlow.openSettings( 'shipping', 'woocommerce-services-settings' );
+
+	// Wait for "Add package" to finish loading, click "edit" once this session is loaded
+	await waitForSelectorAndText('.button:not([disabled])', 'Add package');
+	let editButton = await page.$( '.button.is-compact', { text: 'Edit' } );
+	do {
+		if( editButton ) {
+			await expect( page ).toClick( '.button.is-compact', { text: 'Edit' } );
+			// Delete package
+			await page.waitForSelector( '.packages__add-edit-title.form-section-heading', { text: 'Edit package' } );
+			await expect( page ).toClick( '.button.packages__delete.is-scary.is-borderless', { text: 'Delete this package' } );
+		}
+		editButton = await page.$( '.button.is-compact', { text: 'Edit' } );
+	} while ( editButton );
+};
+
+// Click save and wait until it's saved.
+const saveAndWait = async () => {
```

Comment:
Let's use import `{ saveAndWait } from '../../utils/components';` for this function.